app/src/main/java/org/oppia/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -41,6 +41,9 @@ class AddProfileActivityPresenter @Inject constructor(
   private var allowDownloadAccess = false
   private var inputtedPin = false
   private var inputtedConfirmPin = false
+  private var storyTextSize = 14f
+  private var appLanguage = "English"
+  private var audioLanguage = "No Audio"
```

Comment:
For these & other constants: I feel like we should define them as actual private file-level constants to emphasize that these are profile default values.

Reply:
Done.